languages/go/oso.go

```diff
@@ -152,6 +157,42 @@ func (o Oso) IsAllowed(actor interface{}, action interface{}, resource interface
 	}
 }
 
+/*
+Return a set of actions allowed by the given (actor, resource) combination allowed
+by the policy.
+*/
+func (o Oso) GetAllowedActions(actor interface{}, resource interface{}, allowWildcard bool) (map[interface{}]interface{}, error) {
+	results := make(map[interface{}]interface{})
+	query, err := (*o.p).queryRule("allow", actor, resource)
```

Comment:
This needs to pass in `ValueVariable{"action"}` as the 3rd arg to queryRule.

Reply:
Right, of course. Although that gave me a compiler error, I needed `types.ValueVariable("action")`.